src/DssSpell.sol

```diff
@@ -16,439 +16,114 @@
 
 pragma solidity 0.6.12;
 
-import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
+import {Fileable} from "dss-exec-lib/DssExecLib.sol";
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 import "dss-interfaces/dss/ClipAbstract.sol";
```

Comment:
I think `ClipAbstract` can be removed as not used in the spell

Reply:
Oh, yeah, I thought I would need this when doing the clipper changes, but I just use `Fileable`.